app/models/rubygem.rb

```diff
@@ -136,8 +139,11 @@ def indexed_versions?
   end
 
   def owned_by?(user)
-    return false unless user
-    ownerships.exists?(user_id: user.id)
+    owners.include? user
+  end
+
+  def unconfirmed_ownership?(user)
+    ownerships_including_unconfirmed.unconfirmed(user).exists?
```

Comment:
> `scope :unconfirmed, ->(user) { where("confirmed_at IS NULL").where(user: user) }`

Passing `user` to the scope is making this confusing. As in, why does `unconfirmed` depends on user when `confirmed` scope doesn't.
Lets keep `where(user: user)` right here
```
ownerships_including_unconfirmed.unconfirmed.where(user: user).exists?
```

Reply:
https://github.com/rubygems/rubygems.org/pull/2357#discussion_r452394530
Then it'll not follow this. Is that fine?

Reply:
Yeah, it is fine. I didn't realize at the time if will conflict with our definition of `confirmed`.